apt-*: add more info links; apt-file: add regex search example

patricedenis opened the pull request:
Related: #5510

patricedenis commented:
#5510

patricedenis commented:
I placed manned.org links as I understood we do now in #5143. Or I am supposed to link to the official Debian latest man pages?

bl-ue commented on pages/linux/apt-file.md:
```diff
@@ -1,15 +1,20 @@
 - Search for packages that contain the specified file or path:
 
-`apt-file search {{part/of/filename}}`
+`apt-file {search|find} {{part/of/filename}}`
```
```suggestion
`apt-file {{search|find}} {{part/of/filename}}`
```

bl-ue commented on pages/linux/apt-file.md:
```diff
@@ -1,15 +1,20 @@
 - List the contents of a specific package:
 
-`apt-file list {{package_name}}`
+`apt-file {show|list} {{package_name}}`
```
```suggestion
`apt-file {{show|list}} {{package_name}}`
```

bl-ue commented on pages/linux/apt-file.md:
```diff
@@ -1,15 +1,20 @@
+
+- Search for packages that match the regular expresssion given in `pattern`:
+
+`apt-file {search|find} --regexp {{pattern}}
```
```suggestion
`apt-file {{search|find}} --regexp {{pattern}}
```

bl-ue commented on pages/linux/apt-key.md:
```diff
@@ -1,6 +1,9 @@
 
 > Key management utility for the APT Package Manager on Debian and Ubuntu.
+> Be carefull as `apt-key` is now deprecated except for the use of `apt-key del`
+> in maintainer scripts to remove existing keys from the main keyring.
```
```suggestion
> Note: `apt-key` is now deprecated (except for the use of `apt-key del` in maintainer scripts).
```

patricedenis commented:
For the deprecation of the usage of apt-key, see:

- [](https://www.linuxuprising.com/2021/01/apt-key-is-deprecated-how-to-add.html)

- [](https://wiki.debian.org/DebianRepository/UseThirdParty)

bl-ue commented on pages/linux/apt-key.md:
```diff
@@ -1,6 +1,8 @@
 # apt-key
 
 > Key management utility for the APT Package Manager on Debian and Ubuntu.
+> Be carefull as `apt-key` is now deprecated except for the use of `apt-key del` in maintainer scripts to remove existing keys from the main keyring.
```
```suggestion
> Note: `apt-key` is now deprecated (except for the use of `apt-key del`).
```

bl-ue commented:
@patricedenis will you please also update the translations with this info (at least the links)?  Feel free to [`set-more-info-link.py`](https://github.com/tldr-pages/tldr/blob/ce4846141a0950fb05d467dd6849a311574c06e0/scripts/set-more-info-link.py) for that.  Usage:
```
python3 set-more-info-link.py -p platform/page.md "https://new.link"
# e.g.
python3 set-more-info-link.py -p linux/apt.md "https://manned.org/apt.8"
```
The nice thing about it as that it will add the link to all of the translations of the page, in the [page's language](https://github.com/tldr-pages/tldr/blob/ce4846141a0950fb05d467dd6849a311574c06e0/scripts/set-more-info-link.py#L8-L33).

patricedenis commented:
oh great !

patricedenis commented:
I'm not sure for my history of commit cause I made some commits from Web interface and pull them after that to local git structure which did not catch them as single one but as a whole merge commit. So I split them up after that but the original commit hashes are lost even if I have rewritten the content. 
I'm not sure I clear as well here ^^.

bl-ue commented:
No problem @patricedenis, it really doesn't matter what the commits look like as long as the files look the way we want them to in the end.

bl-ue commented on pages/linux/apt-key.md:
```diff
@@ -1,6 +1,8 @@
 # apt-key
 
 > Key management utility for the APT Package Manager on Debian and Ubuntu.
+> Note: `apt-key` is now deprecated (except for the use of `apt-key del`).
```
I had an old suggestion that suggested this, it it got erased and then I made a new one but forgot to add this:
```suggestion
> Note: `apt-key` is now deprecated (except for the use of `apt-key del` in maintainer scripts).
```

bl-ue commented:
I'll like this PR in #5510

bl-ue commented:
@patricedenis will you please also add the link to [`apt-mark.md`](https://github.com/tldr-pages/tldr/blob/ce4846141a0950fb05d467dd6849a311574c06e0/pages/linux/apt-mark.md)?

patricedenis commented:
> I'll link this PR in #5510

done it already in the first comment ^^

patricedenis commented:
done, I forgot it thanks!

bl-ue reviewed (approved):
Nice good batch of links, thank you @patricedenis.

sbrl reviewed (commented):
Awesome work! Thanks so much for this.

Considering that `apt` is a debian-specific command though, wouldn't The Debian man pages be preferable here? e.g. <https://manpages.debian.org/apt/apt-mark.8.en.html>.

patricedenis commented:
I'm OK with that. I just did'nt get where do we need to apply the manned.org links.
When we have some distribution specific command we ink to the distribution official man pages?

patricedenis commented:
In that case, I should update all the links to the Debian man pages, I checked they're all included.
What do you think?

navarroaxel commented on pages.es/linux/apt.md:
```diff
@@ -2,6 +2,7 @@
 
 > Herramienta de gestión de paquete para distribuciones basadas en Debian.
 > Se recomienda sustituirlo por apt-get cuando se use interactivamente en Ubuntu 16.04 o versiones posteriores.
+> Más información: <https://manpages.debian.org/latest/apt/apt.8.en.html>.
```
```suggestion
> Más información: <https://manpages.debian.org/latest/apt/apt.html>.
```

navarroaxel commented on pages.it/linux/apt-mark.md:
```diff
@@ -1,6 +1,7 @@
 # apt-mark
 
 > Servizio per cambiare lo stato di un pacchetto installato.
+> Maggiori informazioni: <https://manpages.debian.org/latest/apt/apt-mark.8.en.html>.
```
```suggestion
> Maggiori informazioni: <https://manpages.debian.org/latest/apt/apt-mark.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt-cache.md:
```diff
@@ -1,6 +1,7 @@
 # apt-cache
 
 > Buscador de pacotes para distribuições baseadas no Debian.
+> Mais informações: <https://manpages.debian.org/latest/apt/apt-cache.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt/apt-cache.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt-file.md:
```diff
@@ -1,6 +1,7 @@
 # apt-file
 
 > Buscador de arquivos nos pacotes apt, incluindo os não instalados.
+> Mais informações: <https://manpages.debian.org/latest/apt-file/apt-file.1.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt-file/apt-file.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt-get.md:
```diff
@@ -2,6 +2,7 @@
 
 > Gerenciador de pacotes das distribuições baseadas em Debian.
 > Procure por pacotes utilizando o `apt-cache`.
+> Mais informações: <https://manpages.debian.org/latest/apt/apt-get.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt/apt-get.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt-key.md:
```diff
@@ -1,6 +1,7 @@
 # apt-key
 
 > Gerenciador de chaves utilizado pelo gerenciador de pacotes APT nas distribuições baseadas em Debian.
+> Mais informações: <https://manpages.debian.org/latest/apt/apt-key.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt/apt-key.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt-mark.md:
```diff
@@ -1,6 +1,7 @@
 # apt-mark
 
 > Utilitário que altera as configurações dos pacotes instalados.
+> Mais informações: <https://manpages.debian.org/latest/apt/apt-mark.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt/apt-mark.html>.
```

navarroaxel commented on pages.pt_BR/linux/apt.md:
```diff
@@ -1,6 +1,7 @@
 # apt
 
 > Gerenciador de pacotes das distribuições baseadas em Debian.
+> Mais informações: <https://manpages.debian.org/latest/apt/apt.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apt/apt.html>.
```

navarroaxel commented on pages.pt_BR/linux/aptitude.md:
```diff
@@ -1,6 +1,7 @@
 # aptitude
 
 > Gerenciador de pacotes das distribuições baseadas em Debian.
+> Mais informações: <https://manpages.debian.org/latest/aptitude/aptitude.8.en.html>.
```
```suggestion
> Mais informações: <https://manpages.debian.org/latest/aptitude/aptitude.html>.
```

navarroaxel commented on pages.tr/linux/apt-get.md:
```diff
@@ -2,6 +2,7 @@
 
 > Debian ve Ubuntu paket yönetim aracı.
 > Paket aramak için `apt-cache` komutunu kullanın.
+> Daha fazla bilgi için: <https://manpages.debian.org/latest/apt/apt-get.8.en.html>.
```
```suggestion
> Daha fazla bilgi için: <https://manpages.debian.org/latest/apt/apt-get.html>.
```

navarroaxel commented on pages.tr/linux/aptitude.md:
```diff
@@ -1,6 +1,7 @@
 # aptitude
 
 > Debian ve Ubuntu paket yönetim aracı.
+> Daha fazla bilgi için: <https://manpages.debian.org/latest/aptitude/aptitude.8.en.html>.
```
```suggestion
> Daha fazla bilgi için: <https://manpages.debian.org/latest/aptitude/aptitude.html>.
```

navarroaxel commented on pages.zh/linux/apt-file.md:
```diff
@@ -1,6 +1,7 @@
 # apt-file
 
 > 在 apt 软件包中查找文件，其中也包括未安装的软件
+> 更多信息： <https://manpages.debian.org/latest/apt-file/apt-file.1.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt-file/apt-file.html>.
```

navarroaxel commented on pages.zh/linux/apt-cache.md:
```diff
@@ -1,6 +1,7 @@
 # apt-cache
 
 > Debian 和 Ubuntu 的包查询工具.
+> 更多信息： <https://manpages.debian.org/latest/apt/apt-cache.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt/apt-cache.html>.
```

navarroaxel commented on pages.zh/linux/apt-get.md:
```diff
@@ -2,6 +2,7 @@
 
 > Debian 和 Ubuntu 的软件包管理工具.
 > 使用`apt-cache`查找包.
+> 更多信息： <https://manpages.debian.org/latest/apt/apt-get.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt/apt-get.html>.
```

navarroaxel commented on pages.zh/linux/apt-key.md:
```diff
@@ -1,6 +1,7 @@
 # apt-key
 
 > Debian 和 Ubuntu 上的 APT 软件包管理器的密钥管理工具
+> 更多信息： <https://manpages.debian.org/latest/apt/apt-key.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt/apt-key.html>.
```

navarroaxel commented on pages.zh/linux/apt-mark.md:
```diff
@@ -1,6 +1,7 @@
 # apt-mark
 
 > 修改已安装软件包状态的工具
+> 更多信息： <https://manpages.debian.org/latest/apt/apt-mark.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt/apt-mark.html>.
```

navarroaxel commented on pages.zh/linux/apt.md:
```diff
@@ -1,6 +1,7 @@
 # apt
 
 > 基于 Debian 的发行版上的软件包管理工具.
+> 更多信息： <https://manpages.debian.org/latest/apt/apt.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/apt/apt.html>.
```

navarroaxel commented on pages.zh/linux/aptitude.md:
```diff
@@ -1,6 +1,7 @@
 # aptitude
 
 > Debian 和 Ubuntu 上的软件包管理工具.
+> 更多信息： <https://manpages.debian.org/latest/aptitude/aptitude.8.en.html>.
```
```suggestion
> 更多信息： <https://manpages.debian.org/latest/aptitude/aptitude.html>.
```

navarroaxel commented on pages/linux/apt-add-repository.md:
```diff
@@ -1,6 +1,7 @@
 # apt-add-repository
 
 > Manages apt repository definitions.
+> More information: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.html>.
```

navarroaxel commented on pages/linux/apt-get.md:
```diff
@@ -2,6 +2,7 @@
 
 > Debian and Ubuntu package management utility.
 > Search for packages using `apt-cache`.
+> More information: <https://manpages.debian.org/latest/apt/apt-get.8.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/apt/apt-get.html>.
```

navarroaxel commented on pages/linux/apt-key.md:
```diff
@@ -1,6 +1,8 @@
 
 > Key management utility for the APT Package Manager on Debian and Ubuntu.
+> Note: `apt-key` is now deprecated (except for the use of `apt-key del` in maintainer scripts).
+> More information: <https://manpages.debian.org/latest/apt/apt-key.8.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/apt/apt-key.html>.
```

navarroaxel commented on pages/linux/apt-mark.md:
```diff
@@ -1,6 +1,7 @@
 # apt-mark
 
 > Utility to change the status of installed packages.
+> More information: <https://manpages.debian.org/latest/apt/apt-mark.8.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/apt/apt-mark.html>.
```

navarroaxel commented on pages/linux/apt.md:
```diff
@@ -2,6 +2,7 @@
 
 > Package management utility for Debian based distributions.
 > Recommended replacement for apt-get when used interactively in Ubuntu versions 16.04 and later.
+> More information: <https://manpages.debian.org/latest/apt/apt.8.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/apt/apt.html>.
```

navarroaxel commented on pages/linux/aptitude.md:
```diff
@@ -1,6 +1,7 @@
 # aptitude
 
 > Debian and Ubuntu package management utility.
+> More information: <https://manpages.debian.org/latest/aptitude/aptitude.8.en.html>.
```
```suggestion
> More information: <https://manpages.debian.org/latest/aptitude/aptitude.html>.
```

sbrl commented on pages.es/linux/apt-add-repository.md:
```diff
@@ -1,6 +1,7 @@
 # apt-add-repository
 
 > Gestiona las definiciones del repositorio apt.
+> Más información: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.en.html>.
```
```suggestion
> Más información: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>.
```

sbrl commented on pages.de/linux/apt-add-repository.md:
```diff
@@ -1,6 +1,7 @@
 # apt-add-repository
 
 > Editiere die Repository-Listen.
+> Mehr Informationen: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.en.html>.
```
```suggestion
> Mehr Informationen: <https://manpages.debian.org/latest/software-properties-common/apt-add-repository.1.html>.
```

sbrl commented on pages.de/linux/apt-get.md:
```diff
@@ -2,6 +2,7 @@
 
 > Debian und Ubuntu Paket Management Tool.
 > Suche mit `apt-cache` nach Paketen.
+> Mehr Informationen: <https://manpages.debian.org/latest/apt/apt-get.8.en.html>.
```
```suggestion
> Mehr Informationen: <https://manpages.debian.org/latest/apt/apt-get.8.html>.
```

sbrl commented on pages.de/linux/apt.md:
```diff
@@ -2,6 +2,7 @@
 
 > Debian und Ubuntu Paket Management Tool.
 > Empfohlene Alternative zu apt-get seit Ubuntu 16.04.
+> Mehr Informationen: <https://manpages.debian.org/latest/apt/apt.8.en.html>.
```
```suggestion
> Mehr Informationen: <https://manpages.debian.org/latest/apt/apt.8.html>.
```

sbrl commented on pages.es/linux/apt-cache.md:
```diff
@@ -1,6 +1,7 @@
 # apt-cache
 
 > Herramienta de consulta de paquetes para Debian y Ubuntu.
+> Más información: <https://manpages.debian.org/latest/apt/apt-cache.8.en.html>.
```
```suggestion
> Más información: <https://manpages.debian.org/latest/apt/apt-cache.8.html>.
```

sbrl commented on pages.es/linux/apt-file.md:
```diff
@@ -1,6 +1,7 @@
 # apt-file
 
 > Busca archivos en paquetes apt, incluyendo los que aún no fueron instalados.
+> Más información: <https://manpages.debian.org/latest/apt-file/apt-file.1.en.html>.
```
```suggestion
> Más información: <https://manpages.debian.org/latest/apt-file/apt-file.1.html>.
```

sbrl commented:
Haha wow @navarroaxel, that was a lot of comments? Perhaps next time in this instance it might have been better and less overwhelming to leave 1 generic comment?

@patricedenis: I've taken the liberty of making the more information links you've added language-agnostic, if that's ok?

For future reference, the find-in-files feature of GitHub's Atom works splendidly for this.

patricedenis commented:
@sbrl no problemo, thanks !

> For future reference, the find-in-files feature of GitHub's Atom works splendidly for this.

OK, I didn't know the function. I will try it next time. thanks again !

And a big thank you as well to @navarroaxel for the complete review !
